pipelines/_ingredients/ApTemplateCalibrateImage.yaml

```diff
@@ -0,0 +1,80 @@
+description: The AP template building pipeline, with the new CalibrateImageTask.
+# Look in subdirectories of $AP_PIPE_DIR/pipelines to find customized pipelines
+# for each camera. Those pipelines import this general template-building pipeline.
+
+# WARNING: some camera-specific pipelines importing this pipeline presently
+# blow away all the configs that are set in this file.
+# To update a pipeline config prior to DM-35504, you MUST put it in either,
+# e.g., $AP_PIPE_DIR/config/$CAMERA/someTask.py, or in a camera-specific,
+# pipeline, e.g., $AP_PIPE_DIR/pipelines/$CAMERA/ApTemplate.yaml.
```

Comment:
```suggestion
# pipeline, e.g., $AP_PIPE_DIR/pipelines/$CAMERA/ApTemplateCalibrateImage.yaml.
```

Reply:
Just to check: this warning re: DM-35504 still applies here? If we're adding a CalibrateImage override file to `obs_lsst/config/imsim` (which we want, because they're very specific to imsim), does follow the spirit comment given here, go against it, or is it orthogonal to this?

Reply:
Yes, it still applies. The comment is about the pipeline as a whole, and we still have to hack around some overrides for e.g. `makeWarp`.